open-company/open-company-web
# UX desktop: invite Slack button

Card: https://trello.com/c/jcgS60zf
Abstract: https://share.goabstract.com/58526e22-02dc-43dd-aa53-4ff041793aa8

To test:
- signup via email
- go to invite team
- [x] do you see a button to add Slack? Good
- [x] design is ok? Good
- click the button
- [x] are you redirected to Slack auth first step (small set of permissions)? Good
- now go to invite again
- [x] do you still see the same button? Good
- click it again
- [x] are you redirected to Slack auth *second* step (extended set of permissions)? Good
- now go back to invite
- [x] do you have invite via Slack now? Good
